### ISSUE TITLE
Organizations: Add Mechanism to enable Service control policies on the root

### ISSUE THREAD

**Issue by rayterrill** [terraform-providers/terraform-provider-aws#4545]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Need a mechanism to enable Service control policies on the root. It looks like this needs to be done manually before you can successfully apply policies with Organizations (Organizations, Organize accounts, click the Root on the left pane, click Enable under ENABLE/DISABLE POLICY TYPES, Service control policies.

Attempting to add policies without toggling this setting results in this:
```
aws_organizations_policy_attachment.root_FullAccess: Creating...
  policy_id: "" => "p-FullAWSAccess"
  target_id: "" => "r-SECRET"
Releasing state lock. This may take a few moments...

Error: Error applying plan:

1 error(s) occurred:

* aws_organizations_policy_attachment.root_FullAccess: 1 error(s) occurred:

* aws_organizations_policy_attachment.root_FullAccess: error creating Organizations Policy Attachment: PolicyTypeNotEnabledException: This operation can be performed only for enabled policy types.
        status code: 400, request id: 50573131-5866-11e8-a4c8-2f34931e1acc
```

### References

https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies.html#enable_policies_on_root

**Comment by paurullan** [comment]
I stumbled with this same problem. The work-arround is to manually launch policy enabler.
Luckly I found this lines from @jchrisfarris
https://github.com/jchrisfarris/aws-service-control-policies

**Comment by bflad** [comment]
Pull request submitted: https://github.com/terraform-providers/terraform-provider-aws/pull/8588

**Comment by bflad** [comment]
Support for this has been merged via a new `enabled_policy_types` argument on the `aws_organizations_organization` resource (e.g. `enabled_policy_types = ["SERVICE_CONTROL_POLICY"]`). This will be released in version 2.10.0 of the Terraform AWS Provider, likely later today. 👍

**Comment by bflad** [comment]
This has been released in [version 2.10.0 of the Terraform AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#2100-may-10-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
